debian-buster/docker-entrypoint.sh-symbiflow

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source "/opt/conda/etc/profile.d/conda.sh"
```

Comment:
Please, create subdir `debian-buster/symbiflow` and put the "context" for symbiflow images there. So far, no other image  includes any external asset; we managed to avoid it: https://github.com/hdl/containers/blob/main/debian-buster/superprove.dockerfile#L49-L52.

However, I've been thinking about allowing each build to have an optional context, if a directory exists which matches the name of the dockerfile. Do not reference the `symbiflow` subdir in the dockerfiles, though. pyHDLC will take care of copying the Dockerfile into the context.